app/policies/kindergarden_policy.rb

```diff
@@ -0,0 +1,12 @@
+class KindergardenPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+  end
+
+def create?
```

Comment:
we don't want to allow parents create a kindergarden!
I would erase this action

Reply:
removed